Dockerfile

```diff
@@ -0,0 +1,41 @@
+# EDeN Docker container
+#
+# VERSION       0.1.0
+
+FROM ubuntu:14.04
+
+MAINTAINER Björn A. Grüning, bjoern.gruening@gmail.com and Fabrizio Costa
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt-get -qq update && apt-get install --no-install-recommends -y apt-transport-https \
+    python-dev libc-dev python-pip gfortran libfreetype6-dev libpng-dev python-openbabel pkg-config \
+    build-essential libblas-dev liblapack-dev git-core wget software-properties-common python-pygraphviz \
+    libopenbabel-dev swig libjpeg62-dev && \
+    add-apt-repository ppa:bibi-help/bibitools && add-apt-repository ppa:j-4/vienna-rna && \
+    apt-get -qq update && \
+    apt-get install --no-install-recommends -y rnashapes vienna-rna
+
+# requests: system level package has a bug that makes installing from requirements.txt fail
+RUN pip install distribute --upgrade && \
+    pip install "requests==2.7.0" >> install.log
+
+# weblogo installation from requirements.txt complains about missing numpy, so install beforehand
+RUN pip install "numpy==1.8.0" >> install.log
+
+# scikit-learn installation from requirements.txt complains about missing scipy, so install beforehand
+RUN pip install "scipy==0.14.0" >> install.log
+
+# install from local copy of requirements.txt, changes in this file invalidate the cache
+ADD requirements.txt .
+ADD setup.py .
```

Comment:
you're right, not needed anymore

On Wed, Dec 16, 2015 at 4:56 PM, Björn Grüning <notifications@github.com>
wrote:

> In Dockerfile
> <https://github.com/fabriziocosta/EDeN/pull/59#discussion_r47793262>:
>
> > +    apt-get -qq update && \
> > +    apt-get install --no-install-recommends -y rnashapes vienna-rna
> > +
> > +# requests: system level package has a bug that makes installing from requirements.txt fail
> > +RUN pip install distribute --upgrade && \
> > +    pip install "requests==2.7.0" >> install.log
> > +
> > +# weblogo installation from requirements.txt complains about missing numpy, so install beforehand
> > +RUN pip install "numpy==1.8.0" >> install.log
> > +
> > +# scikit-learn installation from requirements.txt complains about missing scipy, so install beforehand
> > +RUN pip install "scipy==0.14.0" >> install.log
> > +
> > +# install from local copy of requirements.txt, changes in this file invalidate the cache
> > +ADD requirements.txt .
> > +ADD setup.py .
>
> Oh this was referring to the setup.py file.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fabriziocosta/EDeN/pull/59/files#r47793262>.
>



-- 
--------------------------------------------------
Daniel Maticzka, Dipl. Inf.
Lehrstuhl für Bioinformatik
Institut für Informatik
Albert-Ludwigs-Universität Freiburg

Tel: (+49) 761 203 97529
Fax: (+49) 761 203 7462
Mail: maticzkd@informatik.uni-freiburg.de
--------------------------------------------------